receiver/googlecloudspannerreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,44 @@
+receivers:
```

Comment:
Can we add tests for multiple config yaml ?

Reply:
Can you explain in more details what are you expecting to see here?